service/history/api/respondworkflowtaskcompleted/workflow_task_completed_handler.go

```diff
@@ -349,7 +349,7 @@ func (handler *workflowTaskCompletedHandler) handleCommand(
 		var failWFTErr workflow.FailWorkflowTaskError
 		if errors.As(err, &failWFTErr) {
 			if failWFTErr.FailWorkflow {
```

Comment:
This should be renamed to `TerminateWorkflow`: https://github.com/temporalio/temporal/blob/f5f182e818240a128ffb9930004548c932ce2a17/service/history/workflow/command_handler.go#L43